src/core/services/TokenService.ts

```diff
@@ -50,17 +52,29 @@ export class TokenServiceImpl implements TokenService {
   public async getSupportedTokens({ network }: GetSupportedTokensProps): Promise<Token[]> {
     const { WETH } = this.config.CONTRACT_ADDRESSES;
     const yearn = this.yearnSdk.getInstanceOf(network);
-
     const supportedTokens = await yearn.tokens.supported();
+    console.log('Yearn Supported Tokens Network: ', supportedTokens);
 
     // TODO: remove fixedSupportedTokens when WETH symbol is fixed on sdk
     const fixedSupportedTokens = supportedTokens.map((token) => ({
       ...token,
-      symbol: token.address === WETH ? 'WETH' : token.symbol,
+      symbol: token.address.toLowerCase() === WETH ? 'WETH' : token.symbol,
```

Comment:
@kibagateaux Added the `.toLowerCase()` due to the previously described issue with the case sensitivity of the `address` key in the `tokensMap` from the Yearn SDK.